stepactions/get-unreleased-bundle/0.1/README.MD

```diff
@@ -0,0 +1,25 @@
+# get-unreleased-bundle stepaction
+
+This StepAction retrieves the highest bundle version from a specified package and channel in the provided FBC fragment. 
+If package name and/or channel name are not specified, default values are used.
```

Comment:
Suggest: `If package name and/or channel name are not specified, the step will determine the default package and/or channel name.`

Reply:
thanks! done